dashboard/app/models/levels/gamelab.rb

```diff
@@ -44,8 +44,6 @@ class Gamelab < Blockly
     debugger_disabled
     pause_animations_by_default
     start_animations
-    auto_run_setup
-    custom_setup_code
```

Comment:
Adding a note that this won't remove these options from .level files or the `properties` column.